frontend/src/components/Account/common/Tabs/Tabs.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import { useHistory, useLocation } from "react-router-dom";
+import Tab from "../Tab/Tab";
+import "./Tabs.scss";
+
+export const Tabs = ({ tabs, defaultTab }) => {
+  const history = useHistory();
+  const location = useLocation();
+  // selected tab by default will be the first tab
+  const [selectedTab, setSelectedTab] = useState(defaultTab);
+  useEffect(() => {
+    const openedTab = tabs.find(
+      tab => location.pathname.indexOf(tab.url) !== -1
+    );
+    if (openedTab) {
+      setSelectedTab(openedTab.id);
+    }
+    const url = openedTab
+      ? location.pathname + location.search
+      : `${tabs[selectedTab].url}`;
+    history.push(url);
+  }, [history]);
+  useEffect(() => {
+    return history.listen(location => {
+      // when clicking back from the browser's back button the visible tab needs to be set again
+      const locationTab = tabs.find(
+        tab => location.pathname.indexOf(tab.url) !== -1
+      );
+      const locationTabId = locationTab ? locationTab.id : 0;
+      setSelectedTab(locationTabId);
+    });
+  }, [history]);
+  const handleTabClick = tab => {
+    setSelectedTab(tab.id);
+    history.push(tab.url);
+    if (tab.clickHandler && typeof tab.clickHandler === "function") {
```

Comment:
Isn't this always going to be a function judging by line 67?

Reply:
removed the function check